model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -59,5 +60,10 @@ data class ScannerConfiguration(
      * A list with the IDs of scan storages that are called to persist scan results. The strings in this list
      * must match keys in the storages map.
      */
-    val storageWriters: List<String>? = null
+    val storageWriters: List<String>? = null,
+
+    /**
+     * A list of globs that match file paths which are to be excluded from scan results.
+     */
+    val ignoreFilePaths: List<String> = NON_LICENSE_FILENAMES
```

Comment:
I deliberately decided against using `filenames` since this is used in `LicenseFilenamePatterns` and these apply only to the filename, while these pattern here apply to the full file path. @mnonnenmacher  Given said, do you still think `ignoreFilePatterns` is your preferred choice? (I'm ok with that, just wanted to double check).

Reply:
That makes sense, but `FilePath` sounds a bit odd to me, what about `ignorePaths` or `ignorePatterns` instead?

Reply:
For example in `PathExclude` we use `pattern` for a similar kind of matching.

Reply:
Convinced, I went for `ignorePatterns` as you suggested.